astropy/io/misc/asdf/tags/coordinates/tests/__init__.py

```diff
@@ -0,0 +1,6 @@
+import pytest
```

Comment:
All these changes would seem to be very suitable to be split out... (in case this review becomes long...)

Reply:
https://github.com/astropy/astropy/pull/9806

Reply:
Resolving since this was done.